pyvista/plotting/renderer.py

```diff
@@ -178,7 +179,16 @@ def camera_position(self, camera_location):
     @property
     def camera(self):
         """Return the active camera for the rendering scene."""
-        return self.GetActiveCamera()
+        camera = self.GetActiveCamera()
+        def zoom(self, value):
+            """Zoom of the camera."""
+            self.Zoom(value)
+        camera.zoom = types.MethodType(zoom, camera)
+        def up(self, vector):
+            """Up of the camera."""
+            self.SetViewUp(vector)
+        camera.up = types.MethodType(up, camera)
```

Comment:
We already have a method on the renderer and plotter to set the view up vector... I'm not sure what's the best way to proceed here.

This monkey path is fine by me (especially for zoom) but perhaps we need to create a new `Camera` class that wraps the camera with all these methods and then move those off the renderer with a deprecation warning

Reply:
Thanks. I am +1 for new `Camera` class. I will consider the design of the new class.

Reply:
>  I will consider the design of the new class

That would be really awesome, thanks @tkoyama010!